permissions/plugin-azure-container-registry-tasks.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "azure-container-registry-tasks"
+github: "jenkinsci/azure-container-registry-tasks-plugin"
+paths:
+- "org/jenkins-ci/plugins/azure-container-registry-tasks"
```

Comment:
This does not correctly represent pom.xml contents, please see this repo's readme.

CC @batmat

Reply:
Sorry, I have already updated the pom file